src/main/java/com/openshift/internal/restclient/model/build/BuildConfigBuilder.java

```diff
@@ -245,9 +255,42 @@ public ISourceStrategyBuilder fromDockerImage(String tag) {
 			this.fromKind = "DockerImage";
 			return this;
 		}
+	}
+
+	class JenkinsPipelineStrategyBuilder implements IJenkinsPipelineStrategyBuilder {
 		
+		private IBuildConfigBuilder builder;
+		private String jenkinsFilePath;
+		private String jenkinsFile;
 		
+		JenkinsPipelineStrategyBuilder(IBuildConfigBuilder builder){
+			this.builder = builder;
+		}
 
-	}
+		@Override
+		public IJenkinsPipelineStrategyBuilder usingJenkinsfile(String file) {
+			this.jenkinsFile = file;
+			return this;
+		}
 
+		@Override
+		public IJenkinsPipelineStrategyBuilder usingJenkinsfilePath(String filePath) {
```

Comment:
This would be fine to call 'usingFilePath' since you are already using the builder.